lib/widget/dropdown.dart

```diff
@@ -247,15 +247,15 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
                 item.icon != null
                     ? iconframework.Icon.fromModel(item.icon!)
                     : SizedBox(
-                  width: widget._controller.size.toDouble(),
+                  width: widget._controller.gap.toDouble(),
                 ),
                 SizedBox(
                   width: item.icon != null
                       ? item.icon!.size != null
-                      ? item.icon!.size == widget._controller.size
+                      ? item.icon!.size == widget._controller.gap
                       ? 10
                       : checkDifference(item.icon!.size!)
-                      : widget._controller.size.toDouble() + 10.0
+                      : widget._controller.gap.toDouble() + 10.0
```

Comment:
this logic runs when the icon size is not specified, so shouldn't it just be 10 pixels only? since the 1st SizeBox above already account for the empty icon space.